stompjs/stompjs.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for stompjs 2.3.x
```

Comment:
`2.3` alone does that. The patch version for `@types/stompjs` will be filled in as `0` when this is published, then `1` the first time it is edited, etc.